CromIAM/src/main/scala/cromiam/server/CromIamServer.scala

```diff
@@ -25,6 +26,12 @@ object CromIamServer extends HttpApp with CromIamApiService with SwaggerService
   override val oauthConfig: SwaggerOauthConfig = configuration.swaggerOauthConfig
 
   def run(): Unit = {
+    log.info(s"Version {}", VersionUtil.getVersion("cromiam"))
+    /*
+    TODO: We're not passing in our actor system so a new one is getting created by akka.http.scaladsl.server.HttpApp.
```

Comment:
ToL: TODOs are probably not the place to put "let's circle round to this at some point" issues.

Can we github it?

Reply:
Filed and linked: https://github.com/broadinstitute/cromwell/issues/3851